guides/common/modules/proc_updating-disconnected-server.adoc

```diff
@@ -0,0 +1,163 @@
+[id="Updating-Disconnected-{project-context}_{context}"]
+= Updating a Disconnected {ProjectServer}
+
+Use this procedure to update your air-gapped {Project} setup where the connected {ProjectServer}, which synchronizes content from CDN, is air gapped from a disconnected {ProjectServer}, to the next minor version.
+
+.Prerequisite
+* Back up your {ProjectServer}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+
+.Procedure on the connected {ProjectServer}
+
+. Ensure that you have synchronized the following repositories in your connected {ProjectServer}:
++
+* {RepoRHEL8BaseOS}
+* {RepoRHEL8AppStream}
+* {RepoRHEL8ServerSatelliteServerProductVersion}
+* {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
```

Comment:
Use this data for connected, they are the same.

Reply:
Correct.

Reply:
Added :+1: